manager/base.py

```diff
@@ -490,7 +490,8 @@ def get_rules_for_cves(cves: list, rh_account=None) -> dict:
     # pylint: disable=singleton-comparison, unsubscriptable-object
     rules_map = {}
     rule_cve_mapping = {}
-    if rh_account is not None:
+    account_exists = rh_account is not None and RHAccount.select().where(RHAccount.id == rh_account).exists()
```

Comment:
```
RHAccount.id == rh_account
```

and 2 lines below

```
RHAccount.name == rh_account
```

indicates some error

+ it's selecting same table twice, should be simplified

Reply:
You are right, it should be the second condition, good catch... The double select should be fixed now